src/interface/src/app/map/map.component.ts

```diff
@@ -388,6 +394,23 @@ export class MapComponent implements AfterViewInit, OnDestroy, OnInit {
   /** Changes which condition scores layer (if any) is shown. */
   changeConditionsLayer(map: Map) {
     this.mapManager.changeConditionsLayer(map);
+    this.updateLegendWithColormap(map, map.config.dataLayerConfig.colormap);
+  }
+
+  private updateLegendWithColormap(map: Map, colormap?: string) {
+    if (colormap == undefined) {
```

Comment:
nit: this can be if (!colormap)

Reply:
!colormap will match if colormap is an empty string, but we want to fetch values for the default colormap ("viridis") in that case.